scripts/release.sh

```
@@ -116,6 +122,259 @@ if [ -z "$GITHUB_REMOTE" ]; then
     exit 1
 fi
 
+# Helper function to check if a tag exists as a release on GitHub
+check_github_release_exists() {
```

Shellcheck (0.11.0)

[error] Couldn't parse this brace group. Fix to allow more checks.

(SC1073)